presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java

```diff
@@ -116,26 +116,15 @@ public void createTopics(String... topics)
 
     public void createTopics(int partitions, int replication, Properties topicProperties, String... topics)
     {
-        checkState(started.get() && !stopped.get(), "not started!");
-
-        ZkClient zkClient = new ZkClient(getZookeeperConnectString(), 30_000, 30_000, ZKStringSerializer$.MODULE$);
-        try {
-            for (String topic : topics) {
-                AdminUtils.createTopic(zkClient, topic, partitions, replication, topicProperties);
-            }
-        }
-        finally {
-            zkClient.close();
-        }
+        // noop
```

Comment:
kafka07 zk api doesn't support those methods I removed. Plus, the topic is automatically created when a message is produced to it.